wes/wes_output_API.json

```diff
@@ -1,330 +1,260 @@
+            "filter_group": "report",
+            "file_purpose": "Analysis file"
         }
     ],
-    "tumor cimac id": [
-        {
-            "filter_group": "alignment/align_recalibrated",
-            "file_path_template": "analysis/align/{tumor cimac id}/{tumor cimac id}_recalibrated.bam",
-            "short_description": "align_recalibrated file",
-            "long_description": "align_recalibrated file align_recalibrated file",
-            "file_purpose": "Miscellaneous"
-        },
-        {
-            "filter_group": "alignment/align_recalibrated_index",
-            "file_path_template": "analysis/align/{tumor cimac id}/{tumor cimac id}_recalibrated.bam.bai",
-            "short_description": "align_recalibrated_index file",
-            "long_description": "align_recalibrated_index file align_recalibrated_index file",
-            "file_purpose": "Miscellaneous"
-        },
-        {
-            "filter_group": "alignment/align_sorted_dedup",
-            "file_path_template": "analysis/align/{tumor cimac id}/{tumor cimac id}.sorted.dedup.bam",
-            "short_description": "align_sorted_dedup file",
-            "long_description": "align_sorted_dedup file align_sorted_dedup file",
-            "file_purpose": "Miscellaneous"
-        },
-        {
-            "filter_group": "alignment/align_sorted_dedup_index",
-            "file_path_template": "analysis/align/{tumor cimac id}/{tumor cimac id}.sorted.dedup.bam.bai",
-            "short_description": "align_sorted_dedup_index file",
-            "long_description": "align_sorted_dedup_index file align_sorted_dedup_index file",
-            "file_purpose": "Miscellaneous"
-        },
-        {
-            "filter_group": "metrics/coverage_metrics",
-            "file_path_template": "analysis/metrics/{tumor cimac id}/{tumor cimac id}_coverage_metrics.txt",
-            "short_description": "coverage_metrics file",
-            "long_description": "coverage_metrics file coverage_metrics file",
-            "file_purpose": "Miscellaneous"
-        },
+    "normal cimac id": [
         {
-            "filter_group": "metrics/target_metrics",
-            "file_path_template": "analysis/metrics/{tumor cimac id}/{tumor cimac id}_target_metrics.txt",
-            "short_description": "target_metrics file",
-            "long_description": "target_metrics file target_metrics file",
-            "file_purpose": "Miscellaneous"
+            "file_path_template": "analysis/align/{normal cimac id}/{normal cimac id}.sorted.dedup.bam",
+            "short_description": "alignment: bam file with deduplicated reads",
+            "long_description": "Aligned reads were sorted and marked duplicates were removed using the Sentieon Dedup tool (https://support.sentieon.com/manual/usages/general/#dedup-algorithm)",
+            "filter_group": "alignment",
+            "file_purpose": "Source file"
```

Comment:
We are considering only the original fastq / bam raw read files are source files atm. This would possibly be `Analysis View` or `Miscellaneous` in case of an intermediate file.